### PR TITLE
add calendar test DO NOT MERGE 

### PR DESCRIPTION
**Pull request by britt6612**
Testing with adding new snapshot

### CODE REVIEW

File: src/js/components/Button/doc.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const doc = Button => {
     color: colorPropType.description(
       'Fill color for primary, label color for plain, border color otherwise.',
     ),
-    disabled: PropTypes.bool
+    disabled: PropTypes.oneOfType([PropTypes.bool, PropTypes.number])
```
Comment (ShimiSun) [inline src/js/components/Button/doc.js:49]: I don't think we want to add the 'number' type to button as discussed on previous threads.

Comment (britt6612) [reply src/js/components/Button/doc.js:49]: I know Im only doing this for testing purposes

Comment (britt6612) [reply src/js/components/Button/doc.js:49]: It wont allow me to add disabled test without it (if only we had it before we would of saw this error)

**Comment by britt6612** [comment]
Closing this PR:

I was testing why snapshots were failing when passing in `dates` to `disabled` 
In order for `disabled` prop to pass it needed to be passed the `date` with timestamp

**Pull request closed without merging**